### PR TITLE
chore: Remove anthos-dpe from owners

### PR DESCRIPTION
**Pull request by olivi-eh**
This PR removes `anthos-dpe` from the owners of this repository, as it's been fully handed over to the corresponding Yoshi team.

Related PRs:
- https://github.com/googleapis/java-container/pull/779
- https://github.com/googleapis/python-container/pull/285
- https://github.com/googleapis/nodejs-cloud-container/pull/563
- https://github.com/googleapis/google-cloud-java/pull/8271
- https://github.com/googleapis/google-cloud-node/pull/3324

### CODE REVIEW

File: .github/CODEOWNERS [hunk 1]
```diff
@@ -5,8 +5,8 @@
...
 
-# @googleapis/yoshi-python @googleapis/anthos-dpe are the default owners for changes in this repo
-*     @googleapis/yoshi-python @googleapis/anthos-dpe
+# @googleapis/yoshi-python @googleapis/yoshi-python are the default owners for changes in this repo
+*     @googleapis/yoshi-python @googleapis/yoshi-python
 
-# @googleapis/python-samples-reviewers @googleapis/anthos-dpe are the default owners for samples changes
-/samples/   @googleapis/python-samples-reviewers @googleapis/anthos-dpe
+# @googleapis/python-samples-reviewers @googleapis/yoshi-python are the default owners for samples changes
+/samples/   @googleapis/python-samples-reviewers @googleapis/yoshi-python
```
Comment (dandhlee) [inline .github/CODEOWNERS:12]:
```suggestion
# @googleapis/python-samples-reviewers is the default owners for samples changes
/samples/   @googleapis/python-samples-reviewers
```

**Review by dandhlee — APPROVED** [review]
One nit, LGTM.

**Review by dandhlee — CHANGES_REQUESTED** [review]
Oops, hold on this might be an autogenerated file.

**Review by dandhlee — CHANGES_REQUESTED** [review]
Please see comment below to properly format CODEOWNERS file for this repo.

File: .github/CODEOWNERS [hunk 2]
```diff
@@ -5,8 +5,8 @@
 # https://help.github.com/en/github/creating-cloning-and-archiving-repositories/about-code-owners#codeowners-syntax
 # Note: This file is autogenerated. To make changes to the codeowner team, please update .repo-metadata.json.
 
-# @googleapis/yoshi-python @googleapis/anthos-dpe are the default owners for changes in this repo
-*     @googleapis/yoshi-python @googleapis/anthos-dpe
+# @googleapis/yoshi-python @googleapis/yoshi-python are the default owners for changes in this repo
+*     @googleapis/yoshi-python @googleapis/yoshi-python
```
Comment (donmccasland) [inline .github/CODEOWNERS:9]: why twice?

**Review by donmccasland — APPROVED** [review]

File: .github/CODEOWNERS (hunk 2 above)
Comment (dandhlee) [reply .github/CODEOWNERS:9]: It's due to the file being templated. The suggestion I put should help get things fixed.

Comment (bourgeoisor) [reply .github/CODEOWNERS:9]: Ohhh I see. That's OwlBot doing its thing, I think I know what's going on--

**Review by dandhlee — APPROVED** [review]
OwlBot behaved! 🦉 LGTM.

**Pull request merged**